src/components/binding/binding-panel.js

```diff
@@ -215,13 +215,20 @@ genotet.BindingPanel.prototype.updateFileListAfterLoading = function() {
     this.selectGenes_[index] = select;
     select.val(track.fileName).trigger('change');
 
-    // Set track fileName
-    select.on('select2:select', function(event) {
-      var fileName = event.params.data.id;
-      this.signal('updateTrack', {
-        trackIndex: index,
-        fileName: fileName
-      });
-    }.bind(this));
+    // Set track fileName and highlight the track hovered in panel.
+    select
+      .on('select2:select', function(event) {
+        var fileName = event.params.data.id;
+        this.signal('updateTrack', {
+          trackIndex: index,
+          fileName: fileName
+        });
+      }.bind(this))
+      .off().on('select2:open', function(event) {
```

Comment:
That is not true. You can always select a specific element. The select2 container created is NOT global but instead local in panel so you should be able to select a single select2 container down from the panel, and that add event listeners to that specific select container. "we would trigger this event when any select2 in the page" is because of incorrect implementation.